proto/ptypes/controller.proto

```diff
@@ -42,6 +42,9 @@ message Volume {
     string frontend = 5;
     string frontendState = 6;
     bool isExpanding = 7;
```

Comment:
Just notice the naming convention for Protobuf field is wrong. It should be `is_expanding` instead of `isExpanding`.

Not sure if we can fix it now though.

Reply:
I knew. And actually you had told me last year.
I am not sure if we can fix it now, either.

Reply:
It's internal GRPC between engine and replica right? It sounds like we can fix it without breaking anything. Need a separate PR though.

Reply:
No. This is the gRPC between manager and engine. But I can fix the naming in replica.proto and syncagent.proto